src/views/Layout.jsx

```diff
@@ -33,45 +33,61 @@ export function Layout({ lists, listPath }) {
 								Sign in to start
 							</button>
 						</div>
-						<div className="hidden lg:flex w-5/12 right-10 bottom-10 absolute justify-end justify-end ">
-							<img src={Groceries} width={400} height={400} alt="groceries" />
+						<div className="hidden lg:flex w-5/12 pt-10 ">
+							<img
+								src={Groceries}
+								width={400}
+								height={400}
+								alt="Illustration of a person with a shopping cart holding a fruit with one hand and a mobile with an open shopping app in the background"
+							/>
 						</div>
 					</div>
 				)}
 			</main>
-			<footer className="w-full fixed bottom-0 lg:flex bg-offWhite text-darkPurple p-4 font-poppins text-xl">
-				<span>
+			<footer className="w-full fixed bottom-0  bg-offWhite text-darkPurple p-4 font-poppins text-xl">
+				<div className="ps-8">
 					<a
 						href="https://github.com/the-collab-lab/tcl-71-smart-shopping-list/tree/main"
-						className="ps-8"
 						aria-label="Source code on GitHub"
 						target="blank"
 					>
 						<i className="fa-brands fa-github pe-2"></i>
 					</a>
-					Built by{' '}
-					<a href="https://www.linkedin.com/in/borjamarticalvo/" target="blank">
-						Borja
-					</a>
-					,{' '}
-					<a
-						href="https://www.linkedin.com/in/celinelecorvaisier/"
-						target="blank"
-					>
-						Céline
-					</a>
-					,{' '}
-					<a
-						href="https://www.linkedin.com/in/judith-kraziewicz/"
-						target="blank"
-					>
-						Judith
-					</a>{' '}
-					and{' '}
-					<a href="https://www.linkedin.com/in/viviana-yanez/" target="blank">
-						Viviana
-					</a>
-				</span>
+					Built by
+					<span>
+						<a
+							href="https://www.linkedin.com/in/borjamarticalvo/"
+							target="blank"
+							className=" hover:underline"
+						>
+							&nbsp;Borja
```

Comment:
This is just nitpicking, but I moved the `&nbsp;` outside the `a` element so on hover the whitespace doesn't get underlined 😝

Reply:
Ty!